core/src/main/scala/org/apache/spark/memory/MemoryManager.scala

```diff
@@ -72,46 +92,62 @@ private[spark] abstract class MemoryManager {
   def acquireUnrollMemory(
```

Comment:
actually, it's more than a synonym. In `StaticMemoryManager` it's required to preserve existing behavior where unrolling doesn't evict all the blocks.